src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts

```diff
@@ -136,34 +158,58 @@ export class SettingsTreeIndicatorsLabel {
 					localize('configuredElsewhere', "Modified elsewhere");
 				this.scopeOverridesLabel.text = scopeOverridesLabelText;
 
-				const prefaceText = element.isConfigured ?
-					localize('alsoModifiedInScopes', "The setting has also been modified in the following scopes:") :
-					localize('modifiedInScopes', "The setting has been modified in the following scopes:");
-				let contentMarkdownString = prefaceText;
-				let contentFallback = prefaceText;
-				for (const scope of element.overriddenScopeList) {
-					contentMarkdownString += `\n- [${scope}](${scope})`;
-					contentFallback += `\n• ${scope}`;
+				let contentMarkdownString = '';
+				let contentFallback = '';
+				if (element.overriddenScopeList.length) {
+					const prefaceText = element.isConfigured ?
+						localize('alsoModifiedInScopes', "The setting has also been modified in the following scopes:") :
+						localize('modifiedInScopes', "The setting has been modified in the following scopes:");
+					contentMarkdownString = prefaceText;
+					contentFallback = prefaceText;
+					for (const scope of element.overriddenScopeList) {
+						const scopeDisplayText = this.getInlineScopeDisplayText(scope);
+						contentMarkdownString += `\n- <a href="${encodeURIComponent(scope)}" aria-label="${getAccessibleScopeDisplayText(scope, this.languageService)}">${scopeDisplayText}</a>`;
```

Comment:
Is there anyway to do this by building DOM elements instead of building HTML strings? It's hard to guarantee that is safe. For example here you could inject HTML via a language pack.

Reply:
Reverting for now. With the revert, the screenreader will read the "Language > Scope" line as-is, just like how a user would read it.